tests/unit/test_cli.py

```diff
@@ -34,6 +34,7 @@ def _common_help_options(result):
     assert "-f, --ruleset-file FILENAME" in result.output
     assert "--debug" in result.output
     assert "--json FILENAME" in result.output
+    assert "--xunit FILENAME" in result.output
```

Comment:
Sorry for confusion but `_common_help_options` is used on more places
and we added that option just to `colin check`.

Therefore test `test_list_checks_help_command` failed
 
https://ci.centos.org/job/user-cont-colin-pr/117/

Moving the assertion to the `test_check_help_command` should fix that